<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_Cliente</class>
 <widget class="QWidget" name="frm_Cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #e8f5e9;
}</string>
  </property>
  <widget class="QPushButton" name="btn_Add">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 2px solid #d1c4b2;
    border-radius: 12px;
    color: #5a5a5a;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px;
    background-image:url(:/iconAdd/Img/adicionar.png); 
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #f2ebe0;
    border-color: #c4b49e;
}

QPushButton:pressed {
    background-color: #e8dfcf; 
    border-color: #b39b8d; 
    padding-left: 12px; 
    padding-top: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_voltar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: #f5f5f5; /* Fundo claro */
    border: 2px solid #cccccc; /* Borda cinza */
    border-radius: 10px; /* Bordas arredondadas */
    padding: 10px; /* Espaçamento interno */
    color: #333333; /* Cor do texto */
    font-size: 14px; /* Tamanho do texto */
	background-image:url(:/iconRet/Img/retornar.png); /* Caminho da imagem */
    background-repeat: no-repeat; /* Evita repetição da imagem */
    background-position: center; /* Centraliza a imagem no botão */
	transition: all 0.3s ease;
}
QPushButton:hover{
    background-color: #e0e0e0; /* Fundo mais claro ao passar o mouse */
    border-color: #aaaaaa; /* Borda mais escura ao passar o mouse */
	transform: scale(1.05); 
}
QPushButton:pressed{
	background-color: #d6d6d6;
	border-color: #888888;
	transform: scale(0.95);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_consul">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff; 
    border: 2px solid #d1c4b2; 
    border-radius: 12px; 
    color: #5a5a5a;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px; 
    background-image:url(:/iconConsu/Img/consultar.png);
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #f2ebe0;
    border-color: #c4b49e;
}

QPushButton:pressed {
    background-color: #e8dfcf;
    border-color: #b39b8d;
    padding-left: 12px;
    padding-top: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_alterar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff; 
    border: 2px solid #d1c4b2; 
    border-radius: 12px; 
    color: #5a5a5a; 
    font-size: 14px;
    font-weight: bold; 
    padding: 10px 16px;
    background-image:url(:/icon_alt/Img/alterar.png);
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #f2ebe0;
    border-color: #c4b49e;
}

QPushButton:pressed {
    background-color: #e8dfcf; 
    border-color: #b39b8d;
    padding-left: 12px;
    padding-top: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_excluir">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffebee; 
    border: 2px solid #ffcdd2;
    border-radius: 10px;
    color: #b71c1c; 
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px;
    background-image:url(:/iconExcluir/Img/excluir.png);
    background-repeat: no-repeat;
    background-position: center;
    padding-left: 40px;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #ffcdd2;
    border-color: #e57373;
    color: #d32f2f; 
}

QPushButton:pressed {
    background-color: #e57373;
    border-color: #b71c1c;
    padding-left: 44px;
    padding-top: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pesquisar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff; 
    border: 2px solid #d1c4b2; 
    border-radius: 12px;
    color: #5a5a5a; 
    font-size: 14px;
    font-weight: bold; 
    padding: 10px 16px;
    background-image: url(:/iconPesq/Img/pesquisar.png); 
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.3s ease; 
}

QPushButton:hover {
    background-color: #f2ebe0; 
    border-color: #c4b49e;
}

QPushButton:pressed {
    background-color: #e8dfcf;
    border-color: #b39b8d; 
    padding-left: 12px; 
    padding-top: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nomeCliente">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Nome do Cliente:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nomeCliente">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_filtro">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff; 
    border: 2px solid #d1c4b2; 
    border-radius: 12px;
    color: #5a5a5a; 
    font-size: 14px; 
    font-weight: bold;
    padding: 10px 16px; 
    background-image:url(:/iconFilt/Img/filtro.png);
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.3s ease; 
}

QPushButton:hover {
    background-color: #f2ebe0; 
    border-color: #c4b49e;
}

QPushButton:pressed {
    background-color: #e8dfcf; 
    border-color: #b39b8d; 
    padding-left: 12px;
    padding-top: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget, QTableView {
    border: 1px solid #dcdcdc; 
    border-radius: 5px; 
    gridline-color: #dcdcdc; 
    background-color: #ffffff; 
    font-size: 14px; 
    color: #333333; 
}

QHeaderView::section {
    background-color: #f0f0f0; 
    color: #333333;
    font-weight: bold; 
    border: 1px solid #dcdcdc; 
    padding: 4px; 
}

QTableWidget::item:selected, QTableView::item:selected {
    background-color: #b3d9ff; 
    color: #000000;
}

QTableCornerButton::section {
    background-color: #f0f0f0; 
    border: 1px solid #dcdcdc;
}

QScrollBar:vertical {
    background: #f0f0f0;
    width: 12px; 
    margin: 2px 0 2px 0; 
    border: none; 
}

QScrollBar::handle:vertical {
    background: #b0bec5; 
    min-height: 20px; 
    border-radius: 6px; 
}

QScrollBar::handle:vertical:hover {
    background: #90a4ae; 
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none; 
    height: 0px; 
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none; 
}

QScrollBar:horizontal {
    background: #f0f0f0; 
    height: 12px; 
    margin: 0 2px 0 2px; 
    border: none;
}

QScrollBar::handle:horizontal {
    background: #b0bec5;
    min-width: 20px; 
    border-radius: 6px;
}

QScrollBar::handle:horizontal:hover {
    background: #90a4ae; 
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none;
    width: 0px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}
</string>
   </property>
   <column>
    <property name="text">
     <string>idCliente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Celular</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cidade</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="icon_adicionar.qrc"/>
  <include location="icon_pesq.qrc"/>
  <include location="icon_filtro.qrc"/>
  <include location="icon_excluir.qrc"/>
  <include location="icon_Consultar.qrc"/>
  <include location="icon_alterar.qrc"/>
  <include location="icon_retornar.qrc"/>
 </resources>
 <connections/>
</ui>
